content/en/docs/releases/supported-releases/index.md

```diff
@@ -18,7 +18,7 @@ on a minor release do not contain backward incompatible changes.
 - [Control Plane/Data Plane Skew](#control-planedata-plane-skew)
 - [Support status of Istio releases](#support-status-of-istio-releases)
 - [Supported releases without known Common Vulnerabilities and Exposures (CVEs)](#supported-releases-without-known-common-vulnerabilities-and-exposures-cves)
-- [Relationship between Istio and Envoy](#supported-envoy-versions)
+- [Supported Envoy Versions](#supported-envoy-versions)
```

Comment:
It's not supported Envoy versions, it's the Envoy version that is consumed within that version.

Reply:
@ericvn 
I see.
So, should I modify the following line in the same way?
https://github.com/akitok/istio.io/blob/update_1.15_EOL_date/content/en/docs/releases/supported-releases/index.md?plain=1#L97

Reply:
To me it would make sense to update the title of the section to match the title of the link. But, should see if there was some discussion in the PR that the section was added in.